netbox/dcim/templatetags/display_region.py

```diff
@@ -0,0 +1,37 @@
+from django import template
+from django.utils.safestring import mark_safe
+
+register = template.Library()
+
+
+def _display_site(obj):
+    """
+    Render a link to the site of an object.
+    """
+    site = getattr(obj, 'site', None)
+    return mark_safe('<a href="{}">{}</a>'.format(site.get_absolute_url(), site)) if site else None
+
+
+@register.simple_tag(takes_context=True)
+def display_region(context, obj, include_site=False):
```

Comment:
IMO we should omit any reference to a site, and limit this tag to only being concerned with rendering the region hierarchy. For example, something like `{% region_tree region %}` might render "US > North Carolina > Raleigh" for the Raleigh region.

In fact, it might be useful to turn this into a general-purpose tag to render other MPTT objects in the same manner.

Reply:
To keep the current scope limited, I'll update the relevant UI and add a simple region tree tag. We can make it generic as a part of a separate FR.

Reply:
I agree on limiting the implementation scope for this FR, but we should give it a generic name now to avoid having to update it later.